src/getters/index.ts

```diff
@@ -782,13 +783,15 @@ export function getDelegationFractionForWallet(
     operator: ParsedOperator,
     { offset = toBN(0) }: { offset?: BN } = {},
 ) {
-    const value = operator.valueWithoutEarnings.plus(offset)
+    const total = operator.operatorTokenTotalSupplyWei
+        .plus(offset)
+        .multipliedBy(operator.exchangeRate)
```

Comment:
Question here: do we rate it before or after applying the offset. I.e. this or
```ts
.multipliedBy(operator.exchangeRate)
.plus(offset)
```

Reply:
I think we should be adding the offset after multiplying by exchange rate as the offset is in DATA

Reply:
Thought so. I'll update!